.travis.yml

```diff
@@ -2,10 +2,12 @@ language: python
 os: linux
 sudo: false
 python:
-- '3.5'
+- '3.6'
```

Comment:
cool switched to python 3.6

Reply:
Yeah 18.04 ships with 3.6 and droplets now supported 18.04 so I decided to switch.